ipa-core/src/net/test.rs

```diff
@@ -465,16 +469,34 @@ impl TestConfigBuilder {
         TestConfig::new(self)
     }
 }
-pub struct TestServer {
+pub struct TestServer<F: ConnectionFlavor = Helper> {
```

Comment:
Yea, it should  be possible to do in a follow up. Designing a new struct from scratch should not require default values for generic types. The story is different for existing types where we need to add a generic parameter. So it is mostly to avoid refactoring all tests that use it and there are quite a few.